_includes/recent-events.html

```diff
@@ -8,60 +8,63 @@
   <div class="grid grid-cols-1 lg:grid-cols-2 gap-4 md:gap-8">
     {% for event in site.events %} {% if event.completed %}
     <a href="{{ event.url }}" class="block">
-    <div
-      class="bg-[{{site.bg-colors.darkBlue}}] text-white md:p-6 p-4 rounded-2xl shadow-lg transition-all duration-300 hover:shadow-xl hover:scale-[1.02] cursor-pointer"
-    >
-      <div class="flex flex-col md:flex-row items-start md:items-center">
-        <!-- Event Image -->
-        <div class="md:w-1/3 w-full mb-4 md:mb-0">
-          <img
-            src="{{ event.banner_image }}"
-            alt="{{ event.title }}"
-            class="w-full h-auto rounded-lg"
-          />
-        </div>
+      <div
+        class="bg-[{{site.bg-colors.darkBlue}}] text-white md:p-6 p-4 rounded-2xl shadow-lg transition-all duration-300 hover:shadow-xl hover:scale-[1.02] cursor-pointer"
+      >
+        <div class="flex flex-col md:flex-row items-start md:items-center">
+          <!-- Event Image -->
+          <div class="md:w-1/3 w-full mb-4 md:mb-0">
+            <img
+              src="{{ event.banner_image }}"
+              alt="{{ event.title }}"
+              class="w-full h-auto rounded-lg"
+            />
+          </div>
 
-        <!-- Event Info -->
-        <div class="md:w-2/3 w-full md:pl-6">
-          <h3 class="md:text-2xl text-base font-bold">{{ event.title }}</h3>
-          <p class="text-sm text-gray-400 italic font-semibold">
-            {{ event.date | date: "%a, %B %e, %Y" }}
-          </p>
+          <!-- Event Info -->
+          <div class="md:w-2/3 w-full md:pl-6">
+            <h3 class="md:text-2xl text-base font-bold">{{ event.title }}</h3>
 
-          <!-- Event Description with Show More/Show Less -->
-          <p class="text-sm my-2">
-            <!-- Shortened version of the description -->
-            <span class="event-description-short"
-              >{{ event.description | truncate: 120, "..." }}</span
-            >
-            <!-- Full description, hidden initially -->
-            <span class="event-description-full hidden"
-              >{{ event.description }}</span
-            >
+            <!-- Event Date Logic -->
+            <p class="text-sm text-gray-400 italic font-semibold">
+              {% if event.end_date and event.date != event.end_date %} {{
```

Comment:
```html
{% if event.end_date and event.date != event.end_date %}
```
can you just do this only? and do let me know.
```html
{% if event.date != event.end_date %}
```

Reply:
logically this should work, but I'm sure that this will not work either. I think the problem arises due to how `date` and `end_date` are being interpreted by jekyll.
![Screenshot 2024-10-10 at 11 01 50 AM](https://github.com/user-attachments/assets/c759e6bf-e3eb-43b6-905f-b6e73a39e9bb)

First convert the date and end_date into Unix Epoch and then compare. If this works, then thats awesome, else just use the conditional logic to check whether the end_date is present or not.

Reply:
this doesnt work
 i just tried it

Reply:
is this ok